src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -99,20 +127,24 @@
           [rn/view {:style style/preview-container}
            [rn/touchable-opacity
             {:active-opacity 1
-             :on-press       #(rf/dispatch [:lightbox/navigate-to-lightbox
-                                            token-id
-                                            {:images [{:image        preview-url
-                                                       :image-width  300 ; collectibles don't have
-                                                                         ; width/height but we need
-                                                                         ; to pass something
-                                                       :image-height 300 ; without it animation
-                                                                         ; doesn't work smoothly and
-                                                                         ; :border-radius not
-                                                                         ; applied
-                                                       :id           token-id
-                                                       :header       collectible-name
-                                                       :description  collection-name}]
-                                             :index  0}])}
+             :on-press       #(rf/dispatch
+                               [:lightbox/navigate-to-lightbox
+                                token-id
+                                {:images                   [{:image        preview-url
+                                                             :image-width  300 ; collectibles don't
```

Comment:
Maybe pass the screen width?

Reply:
I tried, the result isn't good, animation is jumping